contracts/modules/Experimental/TransferManager/LockupVolumeRestrictionTM.sol

```diff
@@ -61,166 +62,218 @@ contract LockupVolumeRestrictionTM is ITransferManager {
     {
     }
 
-
     /** @notice Used to verify the transfer transaction and prevent locked up tokens from being transferred
      * @param _from Address of the sender
      * @param _amount The amount of tokens to transfer
-     * @param _isTransfer Whether or not this is an actual transfer or just a test to see if the tokens would be transferrable
      */
-    function verifyTransfer(address  _from, address /* _to*/, uint256  _amount, bytes /* _data */, bool  _isTransfer) public returns(Result) {
+    function verifyTransfer(address  _from, address /* _to*/, uint256  _amount, bytes /* _data */, bool /*_isTransfer*/) public returns(Result) {
```

Comment:
Can be declared `view`

Reply:
should be complaint with` ITransferManager.sol`